### DISCUSSION TITLE
[ FF7 ] info not displaying

### DISCUSSION THREAD

**Discussion by DLPB2** [julianxhokaxhiu/FFNx#168] [Q&A]
if(gl_draw_text(8, 75, popup_color, (popup_ttl * 255) / POPUP_TTL_MAX, popup_msg))
		{

common.c  probably the issue.

In your implementation it seems X and Y are not absolute but rows and columns?  75 is outside the visible range on my display.  Is it possible to put absolute pixels back as there's no way to calculate the correct placement on screen otherwise when offsetting?

Additionally, the Gamma parameter is not implemented.

**Comment by julianxhokaxhiu** [comment]
Hey,

No sorry this is bgfx debug implementation and I have no control on it. Feel free to approach BGFX author to ask for an improvement around it.

Cheers.

**Comment by DLPB2** [comment]
Hi - there's still a bug though because your current value means some displays won't see the info.

**Comment by julianxhokaxhiu** [comment]
If that's the case, I'll be happy to tackle exact cases, when they will happen. Until now, I didn't had any complain so far.
